subset/security/.htpasswdfail

```diff
@@ -0,0 +1 @@
+admin:$apr1$8Cc9WfFv$pZFQaI1p/XaRiFrrkhQmG.
```

Comment:
potentially can these be generate during the docker build ? or the start_faux script ?